src/components/learner-credit-management/tests/BudgetDetailPage.test.jsx

```diff
@@ -445,6 +448,8 @@ describe('<BudgetDetailPage />', () => {
     expect(mockFetchContentAssignments).toHaveBeenCalledTimes(2); // should be called again on refresh
     expect(mockFetchContentAssignments).toHaveBeenLastCalledWith(expect.objectContaining(expectedTableFetchDataArgs));
     expect(sendEnterpriseTrackEvent).toHaveBeenCalledTimes(1);
+    userEvent.click(viewCourseCTA);
+    expect(sendEnterpriseTrackEvent).toHaveBeenCalledTimes(2);
```

Comment:
nit/curious: why only assert that the Segment event was dispatches on the second click? Would having the event assertion after the first click on line 447 be sufficient without needing the 2nd click?

Reply:
The refreshCTA is also sending a segment event just above it in the same test. I can move the first assertion closer to the call so its easier to map which user event caused the track event to emit.. 👍🏽 

![Screenshot 2023-11-30 at 12 14 00 AM](https://github.com/openedx/frontend-app-admin-portal/assets/82611798/7fb1cbba-5d6a-4b96-85fd-6a11c48b56a3)